lib/ledger.js

```diff
@@ -99,147 +97,278 @@ async function _findAgent() {
-      // store the maintainers DID and make it read-only
-      fs.writeFile(config['veres-one'].maintainerConfigFile,
-        JSON.stringify(bedrock.util.clone(results.maintainerDoc), null, 2),
-        {encoding: 'utf-8', mode: 0o444}, callback);
-
-    }],
     // Store the Board of Governors' DID Document
     // storeGovernorsDoc: ['governorsDoc', (results, callback) =>
     //   fs.writeFile(config['veres-one'].governorsConfigFile,
     //     JSON.stringify(results.governorsDoc.privateDidDocument, null, 2),
     //     'utf-8', callback)],
+
+    // store the maintainers DID and make it read-only
+    await fs.writeFile(config['veres-one'].maintainerConfigFile,
+      JSON.stringify(bedrock.util.clone(maintainerDoc), null, 2),
+      {encoding: 'utf-8', mode: 0o444});
+
     // Build the Veres One Genesis Block Ledger Configuration
-    buildConfig: ['storeMaintainerDoc', /*'storeGovernorsDoc', */
-      (results, callback) => {
-        const ledgerConfig = config['veres-one'].config;
-        const {maintainerDoc} = results;
-        ledgerConfig.ledger = maintainerDoc.doc.id;
-        // const sigValidatorConfig =
-        //   ledgerConfig.ledgerConfigurationValidator[1];
-        // sigValidatorConfig.approvedSigner.push(
-        //   `${maintainerDoc.publicDidDocument.id}#authn-key-1`);
-        callback(null, ledgerConfig);
-      }],
-    sign: ['buildConfig', (results, callback) => {
-      // decrypt the maintainer and governor private keys
-      const {maintainerDoc} = results;
-      const method = maintainerDoc.getVerificationMethod(
-        {proofPurpose: 'capabilityInvocation'});
-      const signingKey = maintainerDoc.keys[method.id];
-      const sign = callbackify(jsigs.sign);
-      return sign(results.buildConfig, {
-        compactProof: false,
-        documentLoader,
-        suite: new Ed25519Signature2018({key: signingKey}),
-        purpose: new AssertionProofPurpose()
-      }, callback);
-    }],
-    create: ['ledgerOwner', 'sign', (results, callback) => {
-      // create Veres One ledger
-      const options = {
-        ledgerConfiguration: results.sign,
-        genesis: true,
-        public: true,
-        owner: results.ledgerOwner.id,
-        plugins: ['veres-one-ticket-service-agent'],
-      };
-      brLedgerAgent.add(results.ledgerOwner, null, options, callback);
-    }]
-  }, err => {
-    if(err) {
-      logger.error('Error while initializing Veres One ledger', {error: err});
-      return callback(err);
-    }
-    callback();
-  });
+    const ledgerConfig = config['veres-one'].config;
+
+    // TODO: Validator currently does not require `test` in UUID documents
+    // if that requriement changes, this code can be used
+
+    // if(config['veres-one-validator'].environment === 'test') {
+    // } else {
+    // }
+
+    const electorPoolId = ledgerConfig.electorSelectionMethod.electorPool =
+    'did:v1:uuid:a3dd75aa-bb78-431d-b767-630831882545';
```

Comment:
I'm wondering if we should hard code this across all Veres One networks... or if it should be autogenerated on startup. Trying to think if there is some sort of attack vector here or accidental network merge/break that happens because everyone is using the same UUID for the elector pool.

Also, people that don't know what's going on are going to get confused by the term "elector" and think there is more power there than there actually is...